CHANGELOG.rst

```diff
@@ -2,6 +2,36 @@
 Release notes
 =============
 
+2.1.0 (2025-09-01)
+==================
+
+Upgrade procedure
+-----------------
+
+.. warning:: Manual intervention required.
+
+    The elastic search mappings are updated to store the publication date (``gepubliceerd_op``) of
+    documents, publications and topics (which is a carbon copy of the ``registratiedatum`` as a convince field)
+    To make sure that this new property contains the ``registratiedatum`` of existing indexed documents either:
+
+    * re-index them from the GPP-publicatiebank
+    * or run the migration script in the container:
+
+        .. code-block:: bash
+
+            python /app/src/manage.py sync_publication_date
+
+Features
+--------
+
+* [:issue:`100`] extend the publication, document and topic documents with extra date fields.
+         All three documents gain the field ``gepubliceerd_op`` and the publication document
+         additionally gains the fields ``datum_begin_geldigheid`` and ``datum_einde_geldigheid``.
+         This feature also includes these fields fields to be included as search parameters in the
+         search endpoint.
+* [:issue:`101`] Altered the field used for sorting the order of the search endpoint from ``laatst_gewijzigd_datum``
+         to the newly added ``gepubliceerd_op`` field.
```

Comment:
```suggestion
* [:issue:`101`] Chronological ordering of search results is now done on the ``gepubliceerd_op`` field
   rather than ``laatst_gewijzigd_datum``.
```